### PR TITLE
website/integrations: Add description for custom enrollment to azure ad

### LINKED ISSUE #8342: How to map user information during enrollment via oauth source
I'm trying to connect Authentik and Microsoft Entra ( former Azure AD) . I can setup the connection and login with my test user but how
do I map the properties from the entra user to the corresponding authentik user?
Any hints or a link to the relevant documentation would be really appreciated.

Maybe this can also be phrased as a more general question how to I map information from federated idp providers. For LDAP there are property mappings but the same does not exist for saml / oauth sources.

### PR DESCRIPTION
**Pull request by tograss**
## Details
Add a description how to setup a custom enrollment flow and mapping of attributes from azure ad closes #8342
---

## Checklist
-   [x] The documentation has been updated
-   [x] The documentation has been formatted (`make website`)

### CODE REVIEW

**Comment by tanberry** [comment]
HI @tograss thanks for the contribution! Can you add an introductory sentence or two to your new section in the docs, explaining what the use case is, why you'd want to do this, etc., please? Then we can get a tech review and hopefully merge soon afterwards.

This also reminds me that I need to go through our Docs and change Azure AD to Entra ID... fun.

**Comment by tograss** [comment]
Hi @tanberry I added a small Introduction .

**Comment by tanberry** [comment]
Thanks so much @tograss . Sounds like a powerful capability. I have one more small edit, then we will get a technical review down by the engineers. Thanks again for your contribution!

**Review by tanberry — CHANGES_REQUESTED** [review]
A couple more small issues, when you have time.

File: website/integrations/sources/azure-ad/index.md [hunk 1]
```diff
@@ -49,3 +49,65 @@ Save, and you now have Azure AD as a source.
 :::note
 For more details on how-to have the new source display on the Login Page see [here](../general#add-sources-to-default-login-page).
 :::
+
+### automatic user enrollment and attribute mapping
+
+Via the following process you can enroll your users without interaction or directly control the mapping azure attribute to authentik
```
[the remark this answers is not in the document]
Comment (tograss) [reply website/integrations/sources/azure-ad/index.md:55]: Looks good to me 👍

**Comment by tanberry** [comment]
OK cool, thanks for reviewing @tograss ! Do you want me to add those edits (I'll be marked as co-contributor), or you (you will get full credit for PR)? I vote you do it and get full credit,  but up to you, I don't know how busy you are...

**Comment by tograss** [comment]
@tanberry Thanks for your suggestion but I'm fine with you co-authoring :-) Thanks again to the whole authentik team for your work.

**Review by tanberry — APPROVED** [review]
Thank so much for these contributions, @tograss!

**Pull request merged**